docs/static_landing_page/index.html

```diff
@@ -8,29 +8,29 @@
 		<meta property="og:title" content="Fairlearn">
 		<meta property="og:type" content="website"/>
 		<meta property="og:description" content="An open source toolkit for assessing and improving fairness in AI">
-		<meta property="og:image" content="_static/images/fairlearn-meta-image.jpg">
+		<meta property="og:image" content="./images/fairlearn-meta-image.jpg">
 		<meta property="og:url" content="http://fairlearn.github.io">
-		<link rel="shortcut icon" type="_static/images/x-icon" href="_static/images/fairlearn-favicon.ico"/>
+		<link rel="shortcut icon" type="./images/x-icon" href="./images/fairlearn-favicon.ico"/>
 	    <!-- Bootstrap CSS -->
 	    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-	    <link rel="stylesheet" type="text/css" href="_static/css/landing_page_style.css">
+	    <link rel="stylesheet" type="text/css" href="./css/landing_page_style.css">
   	</head>
   <body>
 
   	<!-- NAVIGATION -->
 	<nav class="navbar navbar-expand-lg fixed-top">
     	<div class="container py-3">
-      		<img class="img-fluid nav-logo" src="_static/images/fairlearn_full_color.png" alt="">
+      		<img class="img-fluid nav-logo" src="./images/fairlearn_full_color.png" alt="">
       		<button class="navbar-toggler navbar-light" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
         	<span class="navbar-toggler-icon icon-bar"></span>
       		</button>
       		<div class="collapse navbar-collapse" id="navbarResponsive">
 		        <ul class="navbar-nav ml-auto text-center">
 		          <li class="nav-item pr-0 pr-lg-4">
-		            <a class="nav-link" href="https://fairlearn.github.io/user_guide/index.html">User Guide</a>
+		            <a class="nav-link" href="./master/user_guide/index.html">User Guide</a>
```

Comment:
Links like this would need to be updated to point to a different version as the default.